app/Console/Commands/CreateUserAdmin.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Role;
+use App\Models\User;
+use Illuminate\Console\Command;
+
+class CreateUserAdmin extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'user:create-admin {email} {password?}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create Admin User';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $email = $this->argument('email');
+        $password = $this->argument('password');
+        if (! $password) {
+            $password = 'IamAdmin';
+        }
+
+        //если пользов. уже существует, то выдавать сообщение
+        if (User::query()->where('email', $email)->exists()) {
+            $this->error('User already exists');
+
+            return  self::FAILURE;
+        }
+
+        $user = new User();
+        $user->name = 'Admin';
+        $user->email = $email;
+        $user->password = $password;
+        $user->role_id = Role::all()->Where('name', User::IS_ADMIN)->value('id');
```

Comment:
ты сначала достаешь из базы все, а потом фильтруешь в php
эффективней все таки сразу через базу найти 
```suggestion
        $user->role_id = Role::query()->where('name', User::IS_ADMIM)->first()->id;
```